applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/Kconfig

```
@@ -81,6 +81,14 @@ config BT_AUDIO_PRES_DELAY_SRCH_SOURCE
 	  then it will revert to the closest supported value.
 endchoice
 
+config BT_AUDIO_EP_PRINT
+	bool "Print discovered endpoint capabilities"
+	default n
+	help
+	  When debugging it might sometimes be helpful to print the capabilities of the
+	  discovered endpoints.
```

This is an optional nit, but I think it reads a bit better if it just explains what the kconfig does
```suggestion
          Print the supported capabilities of an endpoint when it is discovered.
```